src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_help.py

```diff
@@ -3,103 +3,309 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
+import os.path
+
 from knack.help_files import helps
 
+SERVICE_PRINCIPAL_CACHE = os.path.join('$HOME', '.azure', 'acsServicePrincipal.json')
+
+
+# ACS command help
+
 helps['acs'] = """
      type: group
-     short-summary: Manage Azure Container Services.
+     short-summary: Manage Azure container services.
+"""
+
+helps['acs browse'] = """
+    type: command
+    short-summary: Open a web browser to the dashboard for a container service's orchestrator.
 """
+
+helps['acs create'] = """
+    type: command
+    short-summary: Create a new container service.
+    parameters:
+        - name: --service-principal
+          type: string
+          short-summary: Service principal used for authentication to Azure APIs. If not specified, a new service
```

Comment:
I had tried it out that way, actually, but I balked just because the formatting of a `long-summary:` field in a parameter looked unfamiliar to me:
```
$ az acs create --help

Command
    az acs create: Create a new container service.

Arguments
    --name -n                 [Required]: Name of the container service. You can configure the
                                          default using `az configure --defaults acs=<name>`.
    --resource-group -g       [Required]: Name of resource group. You can configure the default
                                          group using `az configure --defaults group=<name>`.
...
    --service-principal                 : Service principal used for authentication to Azure APIs.
        If not specified, a new service principal with contributor role is created and cached at
        $HOME/.azure/acsServicePrincipal.json to be used by subsequent `az acs` commands.
    --ssh-key-value                     : Configure all linux machines with the SSH RSA public key
                                          string.  Your key should include three parts, for example
                                          'ssh-rsa AAAAB...snip...UcyupgH azureuser@linuxvm.
                                          Default: ~/.ssh/id_rsa.pub.
```
But I agree with the intent of `long-summary:` and I'll make this change.